wafflehaus/edit_response/__init__.py

```diff
@@ -52,27 +52,57 @@ def _replace_lookup(self, replace_str):
             return {}
         return replace_str
 
+    def _foreach(self, conditional, data):
+        if not isinstance(data, list):
+            return data
+        if not all(isinstance(item, dict) for item in data):
```

Comment:
Oh, I wasn't suggesting N-levels, but mixed-content in a single level (i.e. a list with dicts and other other data types). However, that will probably rarely happen and only on poorly defined APIs, because it would be tricky to work with. I'm cool with the way it is.